climada/util/test/test_finance.py

```diff
@@ -199,10 +201,12 @@ def test_pca_CUB_2015_pass(self):
         ref_year = 2015
         cntry_iso = 'CUB'
         res_year, res_val, q = world_bank_wealth_account(cntry_iso, ref_year, no_land=1)
```

Comment:
The variable names `res_` and `ref_` are really hard to distinguish. Could you maybe change the `res_` to `wb_`?